text/0000-rfc-process.md

```diff
@@ -0,0 +1,109 @@
+
+- Buildpack Users - these are developers who want to go from source code to container image
+- Buildpack Authors - these are developers who are building the buildpacks
+- Platform Operator - these are the operators who will be running the buildpack infrastructure
+- Platform Implementor - these are developers who will be implementing the buildpack infrastructure
+- Project Contributor - people contributing to the Cloud Native Buildpacks project
+
+What constitutes a "substantial" change may include the following:
+
+- changes to the spec (buildpack author, platform operator)
+- adding/removing/changing pack commands (buildpack user, buildpack author)
+- lifecycle library interface (platform implementor)
+- process changes (project contributor)
+- governance (project contributor)
+
+If you submit a pull request the team deems warrants an RFC, it will be politely closed with a request for an RFC to be opened first.
+
+## Process
+
+To get an RFC into Cloud Native Buildpacks, first the RFC needs to be merged into the RFC repo. Once an RFC is merged, it's considered 'active' and may be implemented to be included in the project. These steps will get an RFC to be considered:
+
+- Fork the RFC repo: <https://github.com/buildpack/rfcs>
+- Copy `0000-template.md` to `text/0000-my-feature.md` (where 'my-feature' is descriptive. don't assign an RFC number yet).
+- Fill in RFC. Any section can be marked as "N/A" if not applicable.
+- Submit a pull request. The pull request is the time to get review of the proposal from the larger community.
+- Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+
+Once a pull request is opened, the RFC is now in development and the following will happen:
+
+- It will be triaged, labeled, and discussed in a future working group meeting. Working group meetings happen on a weekly cadence barring exceptions. Labeling marks a RFC as general or specific to a subteam.
+- The team will discuss as much as possible in the RFC pull request directly. Any outside discussion will be summarized in the comment thread.
+- When deemed "ready", a team member will propose a "motion for final comment period (FCP)" along with a disposition of the outcome (merge, close, or postpone). This is step taken when enough discussion of the tradeoffs have taken place and the team is in a position to make a decision. Before entering FCP, super majority of the team must sign off.
+- The FCP will last 7 days. If there's unanimous agreement among the team the FCP can close early.
+- For voting, the binding votes are comprised of the core team (and subteam maintainers if labeled as a subteam RFC). Acceptance requires super majority of binding votes in favor. The voting options are the following: Affirmative, Negative, and Abstinence. Non-binding votes are of course welcome.
```

Comment:
What is our threshold for a "super majority"? Do we just mean "majority"?

Reply:
I believe that we'd aimed for super majority to mean 2/3 or greater and no single company can have more than 50% of countable votes.  This latter requirement might mean that a voter might be required to abstain when they otherwise have an opinion.